app/controllers/fileUpload/ValidationErrorsSummaryController.scala

```diff
@@ -51,7 +52,12 @@ class ValidationErrorsSummaryController @Inject()(appConfig: FrontendAppConfig,
       implicit request =>
 
         val schemeName = request.userAnswers.get(SchemeNameQuery).getOrElse("the scheme")
-        val fileDownloadInstructionLink = controllers.routes.FileDownloadController.instructionsFile(chargeType).url
+        val psr = chargeType match {
```

Comment:
Would be more concise to just say val psr = request.userAnswers.get(IsPublicServicePensionsRemedyPage(chargeType, optIndex = None))
Or could move this case match logic into a method on user answers to avoid repeating this logic in several different places

Reply:
This page is also being used by the overseas charge and was timing out if there was a PSR value saved in mongo - will look to moving it into the user answers

Reply:
*timing out if we were in the overseas journey I meant